src/app/views/apps/modals/app-settings-modal.html

```diff
@@ -77,7 +77,7 @@
       <!-- Developer details -->
       <div ng-show="helper.canDeleteApp()">
         <hr>
-        <a class="text-muted" href="" ng-click="app.showDevDetails = !app.showDevDetails"><em> {{ app.showDevDetails && 'Hide' || 'Show'}} {{ 'mno_enterprise.templates.impac.dock.settings.developer_details' | translate }} </em></a>
+        <a class="text-muted" href="" ng-click="app.showDevDetails = !app.showDevDetails"><em> {{ 'mno_enterprise.templates.impac.dock.settings.developer_details.' + (app.showDevDetails && 'hide' || 'show') | translate }} </em></a>
```

Comment:
`(app.showDevDetails && 'hide' || 'show')` would be more readable if you use a ternary operator: `(app.showDevDetails ? 'hide' : 'show')`